lib/output.js

```diff
@@ -632,10 +633,11 @@ function OutputStream(options) {
             }
         }
 
-        comments = comments.filter(comment_filter, node);
+        comments = comments.filter(comment_filter, node).filter(function(c) { return c._dumped !== self; });
```

Comment:
I would like to use arrow function here

Reply:
OK. I once thought this project is using ES5 code only. Then I'll update these lines.

Reply:
Terser uses ES6+ features, like arrow fns, Map, Set, literal strings and so on